src/layouts/editor.vue

```diff
@@ -19,8 +19,9 @@
 
       <v-toolbar-items>
         <v-btn
-          :disabled="editorDirty < 1"
+          :disabled="editorDirty < 1 || isLoading"
           text
+          :loading="isLoading"
           @click="$bus.$emit('editor-save')"
           >Save</v-btn
         >
```

Comment:
we should also disable/hide the "Cancel" button during `isLoading`...as it is right now, if you cancel while save is occurring, the new user still gets created.